### ISSUE TITLE
Greater than (>) in "style" element is being HTML encoded with JRuby

### ISSUE THREAD

**Issue by tbeauvais** [sparklemotion/nokogiri#1315]
If you run the following code in JRuby it will result in the > in the style element being HTML entity encoded &gt;, this does not occur with MRI.

```
html = '<!DOCTYPE html><html><head><style>tr > div { display:block; }</style></head><body><h1>Mr. Belvedere Fan > Club</h1></body></html>'
html_doc = Nokogiri::HTML(html)
puts html_doc.to_html
```

```
<!DOCTYPE html >
<html><head><style>tr &gt; div { display:block; }</style></head><body><h1>Mr. Belvedere Fan &gt; Club</h1></body></html>
```

I have a pull request to address this: https://github.com/sparklemotion/nokogiri/pull/1316

**Comment by flavorjones** [comment]
@tbeauvais - Thanks for pointing this out. I'll take a look at your PR!

**Comment by flavorjones** [comment]
Your PR was merged into master. This will be fixed in 1.6.8 final.

Thank you!
